spec/spec_helper.rb

```diff
@@ -120,7 +120,8 @@
 
   c.allow_http_connections_when_no_cassette = false
   c.default_cassette_options = {
-    :allow_unused_http_interactions => false
+    :allow_unused_http_interactions => false,
+    :decode_compressed_response     => true
```

Comment:
@pkliczewski, @Fryguy I think this would be better to have an override value in a configuration.  The default should be to gzip it, because the general case is that no one looks at the cassettes.  And, then let individual developers override this option locally in a file that isn't checked in.

Not sure exactly where that would live in our configs, but I'm sure someone has a good idea...

cc @chrisarcand @jrafanie @Fryguy @jameswnl